task1/scriptMy.js

```diff
@@ -0,0 +1,57 @@
+// С ниже приведенным объектом решить следующие задачи:
+
+// 1. Создать строку из названий предметов написаных через запятую
+// 2. Посчитать общее количнство студентов и учителей на всех предметах
+// 3. Получить среднее количество студентов на всех пердметах
+// 4. Создать массив из объектов предметов
+// 5. Получить массив из предметов и отсортировать по количеству преподавателей на факультете от большего к меньшему
+
+const subjects = {
+    mathematics: {
+        students: 200,
+        teachers: 6
+    },
+    biology: {
+        students: 120,
+        teachers: 6
+    },
+    geography: {
+        students: 60,
+        teachers: 2
+    },
+    chemistry: {
+        students: 100,
+        teachers: 3
+    }
+}
+
+// 1.
+const neWsubjects = Object.keys(subjects);
```

Comment:
I guess it's strange naming. We should name variables `inThisWay`, also its kinda strange to name keys of subjects as new subjects, i would name like `subjectKeys`. Good naming and following `camelCase` helps a lot when u have tons of code.